### ISSUE TITLE
Bug with CSS class clr

### ISSUE THREAD

**Issue by Aybee** [contao/core-bundle#1287]
Da ist ein Bug, was die Klasse `clr` betrifft.

Online-Demo:
Ein Mitglied bearbeiten und dem Feld "Last name" zur Klasse `w50` die Klasse `clr` hinzufügen. Das Feld "Gender" erscheint dann nicht links, sondern rechts und der Platz links daneben ist leer.

Ursache: Die Klasse `clr` definiert das Widget mit display:table; als Tabelle. Da dieses dann kein Blockelement ist, respektiert es zumindestens im FF die height:80px; nicht. Im FF hat es eine Höhe von 82px.

https://github.com/contao/core-bundle/blob/4.4.12/src/Resources/contao/themes/flexible/src/basic.css#L488

Related https://github.com/contao/core/issues/6093, https://github.com/contao/core/issues/6320

IMHO wäre jetzt die Zeit, die Widgets mal auf Flexitems umzustellen. Das funktioniert eigentlich sehr gut. Hier meine temporäre Lösung.
```css
/* Flexbox Widgets */
.tl_formbody_edit > fieldset,
.subpal,
.subpal .clr:not(.widget) { /* 'inputType' => 'password' */
  display: flex;
  flex-flow: row wrap;
  width: 100%;
}
.widget {
  margin-left: 0;
  margin-right: 0;
  padding-left: 15px;
  padding-right: 15px;
  box-sizing: border-box;
  width: 100%;
}
.w50 {
  width: 50%;
  float: none;
  height: auto;
}
.m12 {
  margin: 0;
  padding-top: 1rem;
}
.clr {
  page-break-before: always;
}
/* Handheld */
@media (max-width:767px) {
  .w50 {
    width: 100%;
  }
  .cbx {
    height: auto;
  }
}
/* Flexbox Widgets - END */
```

**Comment by xchs** [comment]
I also have noticed that there are sometimes problems with the positioning of some form elements in the back end form layout.

* _System_ > _Settings_:

![settings](https://user-images.githubusercontent.com/1218766/34892568-06983886-f7da-11e7-8141-830f18684687.png)

* _Layout_ > _Site structure_:

![site_structure](https://user-images.githubusercontent.com/1218766/34892579-0dff7b8e-f7da-11e7-8920-a2a7f4893539.png)

So, basically everywhere where the form element has the CSS class `cbx`. Increasing the height of the container from `56 px` to `57 px` would fix the problem:
```css
.cbx {
    height: 57px;
}
```

[Contao 4.5.2 / Chrome 63 / Windows 10]

**Comment by leofeyer** [comment]
Instead of changing the implementation, we should analyze the problem and see what causes it. Does the "height 82px" issue occur in FF only?

**Comment by Aybee** [comment]
It's the same with Chrome. You can remove the `display: table;`. That was an old adjustemt that we don't need anymore as there is no `overflow: hidden;` on a widget anymore. See https://github.com/contao/core/issues/6093

`display: table;` on a floated element was bad anyway as a floated element even if it's an inline element automaticaly gets display bock and that not without a reason. You also can not set a height on a table.

Is there an online demo of the latest LTS? If not, can you please set one up? Our demo installations often are full of extensions and experimental stuff as we use them as playground. You can not test this bug in the actual demo 4.5.2 anymore. If there is only one online demo IMHO it should be the LTS.

**Comment by ausi** [comment]
@leofeyer we should probably just backport d51a66eddc98aa0773ccf565ac2231f518354a83 to 4.4 ?

**Comment by leofeyer** [comment]
Fixed in 591ade61a70bb067ba69ab604d67ffb5d261f8c9.
